src/ServiceControl.Audit.Persistence.InMemory/ServiceControl.Audit.Persistence.InMemory.csproj

```diff
@@ -4,18 +4,14 @@
     <TargetFramework>net472</TargetFramework>
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Remove="persistence.manifest" />
-  </ItemGroup>
-
   <ItemGroup>
     <Content Include="persistence.manifest">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
 
-  <ItemGroup Label="ServiceControl compatibility transient dependencies">
```

Comment:
We would like to keep the separate item group with the label to explain why we have the dependency, can you add those back and rebase so we can get this in?